src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -497,6 +498,91 @@ public static IEnumerable<object[]> GetAsync_IPBasedUri_Success_MemberData()
             Assert.True(connectionAccepted);
         }
 
+        [Theory]
+        [InlineData("321.123.456.789")]
```

Comment:
This is kind of weird.  Do you mean for this to be a valid IPv4 address, or not?

Reply:
and similarly below

Reply:
Any non-existent ipv4 endpoint. 

We only care about the traffic through proxy. I think if the endpoint doesn't exist, we can save time connecting to it. I could be wrong.

Do you have any suggestions on generating IPv4 address?

Reply:
Why this should not exist? We should be able to verify CONNECT parameter without actually fetching the content, right? If you really want to try failing connect you can probably use multicast address or address from special blocks (like 127.0.0.127)

Reply:
As @wfurt said, we're not actually using the IPv4 address, so it can be anything.